Mobile/stockdogmobile/components/feedHeader.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import { TouchableOpacity, View, Text } from 'react-native';
+import { Actions } from 'react-native-router-flux';
+import Icon from 'react-native-vector-icons/Feather';
+import styles from '../style/screens/feed';
+
+export default class FeedHeader extends Component {
+
+   constructor(props) {
+      super(props);
+   }
+
+   isChosenHeader = (feedType) => {
```

Comment:
Not a big deal: typically the prefix 'is' for a method signifies that it will return a boolean. I clearer name for this would be something like getHeaderStatusStyle

Reply:
👍